symposion/forms.py

```diff
@@ -11,8 +16,7 @@ class SignupForm(account.forms.SignupForm):
 
     def __init__(self, *args, **kwargs):
         super(SignupForm, self).__init__(*args, **kwargs)
-        del self.fields["username"]
```

Comment:
I'm not sure why usename field is deleted,  but keeping behavior in the patch. L49 delete a field that is not in key_order array.